antrea.yaml

```diff
@@ -0,0 +1,3652 @@
+---
```

Comment:
added by accident?

Reply:
It's automatically added when it's generated by helm.

Reply:
really? this is under the root path. And I executed `make manifest`, never saw a file generated here.

Reply:
ah, sorry, I thought you are talking about the line `---`, this file is added by accident, let me remove it.